src/stratis_cli/_actions/_top.py

```diff
@@ -49,10 +49,10 @@ def create_pool(namespace):
 
         (_, rc, message) = Manager.CreatePool(
            proxy,
-           name=namespace.name,
+           name=namespace.pool_name,
            redundancy=0,
            force=namespace.force,
-           devices=namespace.device
+           devices=namespace.device_name
```

Comment:
This variable name change seems really dubious to me. I don't think it really is the name. It could be a variety of things.